src/components/FlowList.vue

```diff
@@ -1,236 +1,222 @@
+          </PEmptyResults>
+        </template>
+      </p-table>
 
       <template #footer-end>
-        <slot name="footer-end" v-bind="{ page, pages }">
-          <p-pager v-if="pages > 1" v-model:page="page" :pages="pages" />
-        </slot>
+        <p-pager v-if="pages > 1" v-model:page="page" :pages="pages" />
       </template>
     </p-layout-table>
   </div>
 </template>
 
 <script lang="ts" setup>
-  import { media } from '@prefecthq/prefect-design'
+  import { PTable, PEmptyResults, PLink, CheckboxModel, TableColumn } from '@prefecthq/prefect-design'
   import { NumberRouteParam, useDebouncedRef, useRouteQueryParam } from '@prefecthq/vue-compositions'
   import merge from 'lodash.merge'
   import { computed, ref } from 'vue'
-  import { FlowListItem, FlowsDeleteButton, ResultsCount, SearchInput, SelectedCount, FlowsFilterGroup } from '@/components'
-  import { useDeploymentsCount, useFlows, useFlowsFilterFromRoute } from '@/compositions'
-  import { localization } from '@/localization'
+  import {
+    FlowsDeleteButton,
+    LastFlowRun,
+    NextFlowRun,
+    DeploymentsCount,
+    ResultsCount,
+    SearchInput,
+    MiniFlowHistory,
+    SelectedCount,
+    FlowRunTagsInput
+  } from '@/components'
+  import { useCan, useFlowsFilterFromRoute, useWorkspaceRoutes, useOffsetStickyRootMargin, useFlows } from '@/compositions'
+  import { useComponent } from '@/compositions/useComponent'
   import { FlowsFilter } from '@/models/Filters'
+  import { Flow } from '@/models/Flow'
   import { flowSortOptions } from '@/types/SortOptionTypes'
+  import { formatDateTimeNumeric } from '@/utilities/dates'
 
   const props = defineProps<{
     filter?: FlowsFilter,
-    selectable?: boolean,
   }>()
 
-  const emit = defineEmits<{
-    (event: 'delete' | 'update', value?: string): void,
-  }>()
+  const activityTimeSpanInSeconds = 24 * 60 * 60
 
-  const headerExpanded = ref(false)
-  const selected = ref<string[]>([])
+  const { margin } = useOffsetStickyRootMargin()
+  const { FlowMenu } = useComponent()
 
-  const search = ref<string>()
-  const searchDebounced = useDebouncedRef(search, 800)
-  const nameLike = computed({
-    get() {
-      return searchDebounced.value === '' ? undefined : searchDebounced.value
-    },
-    set(value: string | undefined) {
-      searchDebounced.value = value
+  const can = useCan()
+  const routes = useWorkspaceRoutes()
+
+  const flowNameLike = ref<string>()
+  const flowNameLikeDebounced = useDebouncedRef(flowNameLike, 1200)
+  const { filter, clear, isCustomFilter } = useFlowsFilterFromRoute(merge({}, props.filter, {
+    flows: {
+      nameLike: flowNameLikeDebounced,
     },
-  })
+    limit: 50,
+  }))
 
-  const deploymentNameSearch = ref<string>()
-  const deploymentNameLikeDebounced = useDebouncedRef(deploymentNameSearch, 800)
+  const page = useRouteQueryParam('page', NumberRouteParam, 1)
+  const { flows, subscriptions, total, pages } = useFlows(filter, {
+    page,
+  })
```

Comment:
Should we add an interval to this? 
```suggestion
  const { flows, subscriptions, total, pages } = useFlows(filter, {
    page,
    interval: 30000
  })
```